pkg/systemd/overview-cards/systemInformationCard.jsx

```diff
@@ -33,11 +33,22 @@ export class SystemInfomationCard extends React.Component {
         this.state = {};
         this.getDMIInfo = this.getDMIInfo.bind(this);
         this.getMachineId = this.getMachineId.bind(this);
+        this.getSystemUptime = this.getSystemUptime.bind(this);
     }
 
     componentDidMount() {
         this.getDMIInfo();
         this.getMachineId();
+        this.getSystemUptime();
+
+        this.uptimeTimer = setInterval(
+            () => this.getSystemUptime(),
```

Comment:
Just write `setInterval(this.getSystemUptime, 60000)` here. You already bound the function above, so no need to wrap it into yet another function.